components/Marketing/Reasons.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { css } from 'glamor'
+
+import {
+  TeaserFrontTileRow,
+  TeaserFrontTile,
+  fontStyles,
+  Editorial,
+  Button,
+  mediaQueries
+} from '@project-r/styleguide'
+
+import { Link } from '../../lib/routes'
+
+const Reasons = ({ t }) => {
```

Comment:
The spacing of this component should eventually be optimised, especially on mobile. I recommend going for a custom grid and getting ride of `TeaserFrontTileRow`.